converters/google/convert.go

```diff
@@ -39,11 +39,13 @@ var ErrDuplicateAsset = errors.New("duplicate asset")
 // Asset contains the resource data and metadata in the same format as
 // Google CAI (Cloud Asset Inventory).
 type Asset struct {
-	Name      string         `json:"name"`
-	Type      string         `json:"asset_type"`
-	Resource  *AssetResource `json:"resource,omitempty"`
-	IAMPolicy *IAMPolicy     `json:"iam_policy,omitempty"`
-	OrgPolicy []*OrgPolicy   `json:"org_policy,omitempty"`
+	Name            string             `json:"name"`
+	Type            string             `json:"asset_type"`
+	Resource        *AssetResource     `json:"resource,omitempty"`
+	IAMPolicy       *IAMPolicy         `json:"iam_policy,omitempty"`
+	OrgPolicy       []*OrgPolicy       `json:"org_policy,omitempty"`
+	OrgPolicyPolicy []*OrgPolicyPolicy `json:"v2_org_policies,omitempty"`
```

Comment:
Would it make sense to call this V2OrgPolicies to match config-validator? I would generally prefer to be consistent.